packages/functional-tests/lib/targets/remote.ts

```diff
@@ -1,20 +1,14 @@
 import { BaseTarget, Credentials } from './base';
-import { EmailHeader, EmailType } from '../email';
-
 export abstract class RemoteTarget extends BaseTarget {
   async createAccount(
     email: string,
     password: string,
-    options?: any
+    options = { preVerified: 'true' }
   ): Promise<Credentials> {
-    const creds = await this.authClient.signUp(email, password, options || {});
-    const code = await this.emailClient.waitForEmail(
-      email,
-      EmailType.verify,
-      EmailHeader.verifyCode
-    );
-    await this.authClient.verifyCode(creds.uid, code);
-    await this.emailClient.clear(email);
```

Comment:
Is it okay to remove this logic so that the functionality more closely resembles what is possible in the method with the same name in local.ts?

Reply:
@vbudhram would you mind checking https://github.com/mozilla/fxa/pull/16739/commits/4cc6ec614151cfef2f63053c3ad62a13ede63275? I want to make sure I understood FXA-9550 properly.